README.md

```diff
@@ -78,8 +78,8 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-- [x] Add the rockets and missions at separate components
-- [x] Install the react-router-dom and use the version@6.👌
+- [x] Add the functionality for the reversed rocket.
+- [x] Identify which button will appear depend on the active of the rocket👌
```

Comment:
Great job updating your ReadMe

Reply:
- Thanks  @momotsow 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊